tests/flow/test_stress.py

```
@@ -1,195 +1,190 @@
-            conn = self.env.getConnection()
-            conn.ping()
-            conn.close()
-
-            # close the connection pool
-            await pool.aclose()
-
-        asyncio.run(run(self))
+        n_tasks           = 10000 # number of tasks to run
+        n_creations       = 0.5
+        n_node_deletions  = 0.75
+        n_edge_deletions  = 1
```

Ruff (0.12.2)

Local variable `n_edge_deletions` is assigned to but never used

Remove assignment to unused variable `n_edge_deletions`

(F841)